packages/world/test/Spawn.t.sol

```diff
@@ -69,6 +69,12 @@ contract SpawnTest is BiomesTest {
     ReversePosition.set(spawnCoord.x, spawnCoord.y, spawnCoord.z, entityId);
     ObjectType.set(entityId, AirObjectID);
 
+    {
```

Comment:
maybe for later, but we could deploy an all-air chunk for tests instead of setting each air entity manually

Reply:
yea this one lets do in separate PR